pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
```

Comment:
But yesod *is* the framework right ? The website literally says 'yesod web framework'. You can frame it in a different way if you want, just don't use the word `yesod`.